README.md

```diff
@@ -108,6 +108,7 @@ include any necessary hooks in a file called `custom.py` for Python models or `c
         - `transformer` deserialized transformer loaded by DRUM or `load_model` 
         - `y` defaults to None, and will not be used for unsupervised downstream problems. Contains the target labels. Transform can simply pass the raw values on, may transform the values (i.e perform a log transform), 
         or use the `y` values in the transformation of `X` if needed.
+        - As of version 1.4.9, the transform hook need not return `y` if no changes are made to the target. `y` is still a required argument to this hook for custom transforms.
```

Comment:
why is y required?

Reply:
can we just check if the transform wants y?

Reply:
I guess by "required" I mean "we're going to pass y to this hook, but you don't actually need to do anything with it". Does it make sense to not have that requirement? I can check the number of args expected by the method and just pass in y if it wants three/has a `y` arg

Reply:
...changed to check what transform takes